src/select.js

```diff
@@ -216,7 +216,22 @@ export default class CanopySelect extends React.Component {
         if (option.separator) {
           return <li key={`separator${index}`} className="separator" />
         } else {
-          return <li key={option.key} className={selectedIndex === index ? '+selected' : ''} onMouseDown={this.selectItem.bind(this, index)}><a style={option.value !== null ? {} : {color: "rgba(0,0,0,0)"}}>{option.value !== null ? option.value : "null"}</a></li>
+          const linkStyles = {
+            color: option.value !== null ? "" : "rgba(0,0,0,0)",
```

Comment:
what's `rgba(0,0,0,0)` for? that would be black with `0` alpha

Reply:
you could also just say `transparent`

Reply:
that was part of the existing code. not sure why it was written that way.

Reply:
🤷‍♂